docs/readmes/orc8r/dev_minikube.md

```diff
@@ -120,6 +138,26 @@ This section describes how to install based on local charts. However, you can al
 - Stable: <https://artifactory.magmacore.org/artifactory/helm/>
 - Test: <https://artifactory.magmacore.org/artifactory/helm-test/>
 
+Optional: install the fluentd charts by editing the file at `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r/charts/logging/values.yaml`
+
+```yaml
+fluentd_daemon:
+  create: true
+
+  image:
+    repository: IMAGE_REGISTRY_URL
+    tag: IMAGE_TAG
+    pullPolicy: IfNotPresent
+
+  env:
+    elastic_host: "host.minikube.internal"
+    elastic_port: "9200"
+    elastic_scheme: "http"
+```
+
+and equivalent edits for `fluentd_forward`.
```

Comment:
~~It is clearer to _not_ split the files to be edited, i.e. naming one before and one after the edit block. Either name both before (preferred) or both after.~~

Reply:
This can be confusing. Rephrase to be more clear or five a full example with both blocks.

Reply:
Yeah, you're right. I added the block for fluent_forward.